bindings/Fortran/modules/adios2_adios_init_mod_mpi.F90

```diff
@@ -7,9 +7,9 @@
 !
 
 #ifdef ADIOS2_HAVE_FORTRAN_SUBMODULES
-# define ADIOS2_MODULE_PROCEDURE module &
+# define ADIOS2_MODULE_SUBROUTINE module subroutine
 #else
-# define ADIOS2_MODULE_PROCEDURE
+# define ADIOS2_MODULE_SUBROUTINE subroutine
```

Comment:
Seems like an opportunity to consolidate this repeated snippet?

Reply:
Perhaps, though I'd rather save re-structuring for the sake of cleanup to a separate issue